_automating-configurations/workflow-steps.md

```diff
@@ -75,9 +75,13 @@ You can include the following additional fields in the `user_inputs` field if th
 
 You can include the following additional fields in the `previous_node_inputs` field when indicated.
 
-|Field	|Data type	|Description	|
-|---	|---	|---	|
-|`model_id`	|String	|The `model_id` is used as an input for several steps. As a special case for the Register Agent step type, if an `llm.model_id` field is not present in the `user_inputs` and not present in `previous_node_inputs`, the `model_id` field from the previous node may be used as a backup for the model ID.	|
+| Field	          |Data type	| Description	                                                                                                                                                                                                                                                                                               |
+|-----------------|---	|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `model_id`	     |String	| The `model_id` is used as an input for several steps. As a special case for the Register Agent step type, if an `llm.model_id` field is not present in the `user_inputs` and not present in `previous_node_inputs`, the `model_id` field from the previous node may be used as a backup for the model ID.	 |
```

Comment:
For model_id we say "several steps" but then for the new additions we are very explicit on exactly which steps.  We should be consistent with all of them.

Reply:
updated